CompatBot/Database/BotDb.cs

```diff
@@ -45,6 +45,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Stats>().HasIndex(s => new { s.Category, s.Key }).IsUnique().HasName("stats_category_key");
             modelBuilder.Entity<Kot>().HasIndex(k => k.UserId).IsUnique().HasName("kot_user_id");
             modelBuilder.Entity<Doggo>().HasIndex(d => d.UserId).IsUnique().HasName("doggo_user_id");
+            modelBuilder.Entity<ForcedNickname>().HasIndex(d => new { d.UserId, d.GuildId }).IsUnique().HasName("forced_nickname_user_id_guild_id");
```

Comment:
You will also need to remove current migration and regenerate it with the new PK instead of having one useless migration that would never be used, and can cause issues in the process